xchange-stream-coinbasepro/src/test/java/info/bitrich/xchangestream/coinbasepro/CoinbaseProCacheTest.java

```diff
@@ -0,0 +1,358 @@
+package info.bitrich.xchangestream.coinbasepro;
```

Comment:
Please give the tests a bit more meaningful names that would hint what is tested

Reply:
changed